.github/workflows/pr-checker.yaml

```
@@ -0,0 +1,58 @@
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies (if needed)
+        run: pip install re  # Install Python's regex library (not needed if using built-in)
+
+      - name: Check PR Description and Title
+        id: check_pr_details
+        run: |
+          python -c "
+import re
```

actionlint

could not parse as YAML: yaml: line 27: could not find expected ':'

(syntax-check)